ci/linkcheck.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then
+  echo "Doing full link check."
+  FLAGS=""
+else
+  echo "Checking files changed in commit range: $TRAVIS_COMMIT_RANGE"
+  CHANGED_FILES=$(git diff-tree --no-commit-id --name-only -r $TRAVIS_COMMIT_RANGE | tr '\n' ' ')
```

Comment:
Won't this break when running linkcheck locally? It would be nice to use `$(git merge-base HEAD master)` when `TRAVIS_COMMIT_RANGE` is unset.